getmoro/moro
# Authentication updates (frontend + google auth + cleanups) + graphql field validations

To test this:

Make your own version of .env in the backend folder. (use .env.sample file)
Start frontend and backend using `npm start` in each folder and go to `localhost:4000/login`.

When testing the "forgot password", because we don't have an email service yet, you can see the token logged in the backend console.

To test google login, you will need to contact me and tell me your Gmail address. This is because this client is not yet approved by google. (Alternatively, you can create an oauth2 key in your own google console, and provide the client key in .env files of frontend and backend)
